generators/cli/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = generators.Base.extend({
       this.fs.copyTpl(
         this.templatePath('cli.js'),
         this.destinationPath('lib/cli.js'), {
-          pkgSafeName: _.camelCase(this.options.name),
+          pkgSafeName: _.camelCase(this.appname),
```

Comment:
I don't think it should be close. We have (or should have) access to the app name without requiring someone to manually pass an option.